logger/logger_test.go

```diff
@@ -26,7 +26,11 @@ func TestMeta(t *testing.T) {
 		IndexMeta:  true,
 	}
 
-	myLogger := CreateLogger(options, key)
+	myLogger, err := CreateLogger(options, key)
+	if err != nil {
```

Comment:
would change these to t.Fatal here and below

Reply:
good catch, this was missing the latest changes - fixed!